dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -0,0 +1,41 @@
+class LearningGoalEvaluationsController < ApplicationController
+  authorize_resource
+
+  def create
+    learning_goal_evaluation = LearningGoalEvaluation.new(learning_goal_evaluation_params)
+    learning_goal_evaluation.teacher_id = current_user.id
+
+    learning_goal_evaluation.save!
+
+    render json: learning_goal_evaluation
+  end
+
+  def update
+    params.require(:id)
+    learning_goal_evaluation = LearningGoalEvaluation.find_by_id(params[:id])
+
+    if learning_goal_evaluation&.teacher_id == current_user.id
+      if learning_goal_evaluation.update(learning_goal_evaluation_params)
+        render json: learning_goal_evaluation
+      else
+        head :bad_request
+      end
+    else
+      head :not_found
```

Comment:
@etaderhold -- were you suggesting using both `:not_found` and `:forbidden` here or do you think `:not_found` is enough?

Reply:
I think `:not_found` is enough for these two cases ("record doesn't exist" and "user can't view record"). I was merely suggesting to split out "failed to save record with the given parameters" into a separate response code.